selfdrive/controls/lib/longitudinal_planner.py

```diff
@@ -127,19 +129,26 @@ def update(self, sm):
     reset_state = reset_state or not v_cruise_initialized
 
     # No change cost when user is controlling the speed, or when standstill
-    prev_accel_constraint = not (reset_state or sm['carState'].standstill)
+    prev_accel_constraint = not (reset_state or CS.standstill)
 
     if self.mpc.mode == 'acc':
+      # Update VehicleModel
+      LP = sm['liveParameters']
+      x = max(LP.stiffnessFactor, 0.1)
+      sr = max(LP.steerRatio, 0.1)
+      self.VM.update_params(x, sr)
+      current_curvature = self.VM.calc_curvature(math.radians(CS.steeringAngleDeg - LP.angleOffsetAverageDeg), CS.vEgo, LP.roll)
```

Comment:
I believe controlsState has this value, but let's only implement the offset compensation here.

Reply:
Ah yes, I forgot it was published there.